jericho/jericho.py

```diff
@@ -34,6 +34,57 @@
 dlclose_func = CDLL(None).dlclose  # This WON'T work on Win
 dlclose_func.argtypes = [c_void_p]
 
+from collections import defaultdict
+import multiprocessing as mp
+from typing import Iterable, Callable
+import itertools
+
+
+def chunk(iterable: Iterable, n: int, fct: Callable = lambda e: e) -> Iterable[Iterable]:
+    it = iter(iterable)
+    while True:
+        try:
+            first = next(it)
+        except StopIteration:
+            return
+
+        rest = itertools.islice(it, n - 1)
+        yield fct(itertools.chain((first,), rest))
+        # Make sure rest is exhausted
+        for _ in rest:
+            pass
+
+def init_worker(story, seed):
+    worker.env = FrotzEnv(story, seed)
+
+
+def worker(data):
```

Comment:
Just for testing. I don't think there's any reason to keep this non-C parallelized version.